parsing/dialects/dialect1/parserImpls.ftl

```diff
@@ -1792,6 +1792,47 @@ SqlNode SqlSelectTopN(SqlParserPos pos) :
     }
 }
 
+SqlNode InlineCaseSpecific() :
+{
+    SqlNode value;
+    boolean not = false;
+}
+{
+    (
+        value = StringLiteral()
+    |
+        LOOKAHEAD( CompoundIdentifier() <LPAREN> [ <NOT> ] <CASESPECIFIC> <RPAREN> )
+        value = CompoundIdentifier()
+    )
+    <LPAREN>
+    [ <NOT> { not = true; } ]
+    <CASESPECIFIC>
```

Comment:
It looks like in the docs that CASESPECIFIC can also be abbreviated to CS which could be good to add in for completeness.

Reply:
Nice catch, will add.